lambda_services/job_service/job_service.py

```diff
@@ -4,7 +4,7 @@
 import json
 import logging
 import boto3
-from launcher import pdb2pqr_runner, apbs_runner
+from .launcher import pdb2pqr_runner, apbs_runner
```

Comment:
Prepending with `.` may cause an ImportError within AWS. However, adding `job_service.py`'s directory to my PythonPath helped alleviate the Pytest errors I encountered without the `.`

Reply:
Can you test whether or not the `.` causes a problem? The `.` follows the Python3 PEP standard for relative includes and the VS Code flags it as a warning if the `.` is missing.